_posts/2021-05-04-hhvm-4.108.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: "HHVM 4.108"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.108 is released!
+HHVM 4.103&ndash;4.107 remain supported, as do the 4.80 and 4.102 LTS releases.
+
+# Highlights
+
+- The
+  [Hack Standard Library](https://docs.hhvm.com/hack/getting-started/the-standard-library)
+  is now a built-in part of HHVM. Hack projects can now call any HSL functions
+  without downloading and installing any extra packages.
+  - This includes
+    [HSL IO](https://docs.hhvm.com/hack/getting-started/input-and-output),
+    which is no longer considered experimental. Other parts of
+    [hsl-experimental](https://github.com/hhvm/hsl-experimental/)
+    are not yet included.
```

Comment:
Does this look good?

I'll also update the documentation that we link to, since it contradicts some of this.

Reply:
ah, yeah, that probably needs some infra changes to deal with built-in HSL too

Reply:
And yep, this text looks good